client/App.js

```diff
@@ -1,44 +1,33 @@
-//import dependencies
+// import dependencies
 import React from 'react';
 import { render } from 'react-dom';
```

Comment:
Is this being used?

Reply:
yes It is  `render(App, document.getElementById('root'));`